### ISSUE TITLE
this.isSafeJSONString is not a function

### ISSUE THREAD

**Issue by tristobal** [kjur/jsjws#28]
Using you library and trying to verify a signed token using

```javascript
    var sJWS = "the token"
    var sPemX509Cert = "-----BEGIN CERTIFICATE----- etc"
    var jws = new KJUR.jws.JWS();
    var result = jws.verifyJWSByPemX509Cert(sJWS, sPemX509Cert);
```

The console throws the following error:
```javascript
TypeError: this.isSafeJSONString is not a function
    at parseJWS (http://localhost:8100/lib/jsjws/jws-3.2.js:115:13)
    at verifyJWSByPemX509Cert (http://localhost:8100/lib/jsjws/jws-3.2.js:203:7)
    ...
```
I solved creating the mentioned function (from your source code, jws-3.2.js):
```javascript
    //variables
    var jws = new KJUR.jws.JWS();
    jws.isSafeJSONString = function(s, h, p) {
        var o = null;
        try {
    	o = jsonParse(s);
    	if (typeof o != "object") return 0;
    	if (o.constructor === Array) return 0;
    	if (h) h[p] = o;
    	return 1;
        } catch (ex) {
    	return 0;
        }
    };
    var result = jws.verifyJWSByPemX509Cert(sJWS, sPemX509Cert);
```

**Comment by Upperstage** [comment]
This demo shows same problem:
http://kjur.github.io/jsjws/sample_generate.html

**Comment by kjur** [comment]
Thank you for the report. I've fixed them in the latest 3.2.2 release. Thanks.
